### PR TITLE
EvalAI Documentation Grammar Fixes 

### PR DESCRIPTION
**Pull request by MashyC**
Just a few changes that I had sent back to me, that have now been fixed.

### CODE REVIEW

**Comment by deshraj** [comment]
@MashyC can you please resolve the conflicts?

**Comment by deshraj** [comment]
@MashyC the conflicts are not resolved so I won't be able to review the pull request.

**Comment by MashyC** [comment]
I resolved the conflicts.

File: docs/source/architecture_decisions.md [hunk 1]
```diff
@@ -32,26 +33,29 @@ We use underscore **_** in URL patterns.
 
 ### Processing submission messages asynchronously
 
-When a submission message is made, a REST API is called which saves the data related to the submission in the database. A submission involves the processing and evaluation of `input_file`. This file is used to evaluate the submission and then decide the status of the submission, whether it is _FINISHED_ or _FAILED_.
```
Comment (deshraj) [inline docs/source/architecture_decisions.md]: Please remove this new line.

File: docs/source/architecture_decisions.md [hunk 2]
```diff
@@ -32,26 +33,29 @@ We use underscore **_** in URL patterns.
...
 
 ### Submission Worker
 
 The submission worker are responsible for processing submission messages. It listens on a queue named `submission_task_queue`, and on receiving a message for a submission, it processes and evaluates the submission.
 
-One of the major design changes that we decided to implement in the submission worker was to load all the data related to the challenge in the worker's memory, instead of fetching it every time a new submission message arrives. So the worker, when starting, fetches the list of active challenges from the database and then loads it into memory by maintaining the map `EVALUATION_SCRIPTS` on challenge id. This was actually a major performance improvement.
+One of the major design changes that we decided to implement in the submission worker was to load all the data related to challenge in the memory of the worker, instead of fetching it every time whenever a submission message is there for any challenge. So the worker, when starting, fetches the list of active challenges from the database and then loads it into memory by maintaining a map `EVALUATION_SCRIPTS` on challenge id. This was actually a major performance improvement.
```
Comment (deshraj) [inline docs/source/architecture_decisions.md]: Please revert this.

File: docs/source/architecture_decisions.md [hunk 3]
```diff
@@ -32,26 +33,29 @@ We use underscore **_** in URL patterns.
...
 
-Another major design change that we incorporated here was to dynamically import the challenge module and to load it in the map instead of invoking a new python process every time a submission message arrives. So now whenever a new message for a submission is received, we already have its corresponding challenge module being loaded in a map called `EVALUATION_SCRIPTS`, and we just need to call
+Another major design change that we incorporated here was dynamically importing the challenge module and loading it in the map instead of invoking a new Python process every time a submission message arrives. So now, whenever a new message for a submission is received, we already have its corresponding challenge module being loaded in a map `EVALUATION_SCRIPTS`, and we just need to call
```
Comment (deshraj) [inline docs/source/architecture_decisions.md]: Please revert this.

File: docs/source/architecture_decisions.md [hunk 4]
```diff
@@ -32,26 +33,29 @@ We use underscore **_** in URL patterns.
...
 
 ```
 EVALUATION_SCRIPTS[challenge_id].evaluate(*params)
 ```
 
 This was again a major performance improvement, which saved us from the task of invoking and managing Python processes to evaluate submission messages. Also, invoking a new python process every time for a new submission would have been really slow.
+
```
Comment (deshraj) [inline docs/source/architecture_decisions.md]: No need for an extra line.

File: docs/source/challenge_creation.md [hunk 5]
```diff
@@ -2,8 +2,10 @@
 
 One can create a challenge in EvalAI using either:
 
-1. zip configuration
-2. web interface
+
+* Challenge creation using zip file configuration
+
+* Challenge creation using a web interface
```
Comment (deshraj) [inline docs/source/challenge_creation.md]: Please revert these changes.

File: docs/source/directory_structure.md [hunk 6]
```diff
@@ -22,6 +22,7 @@ This app serves users who want to take part in any challenge. It contains code f
 
 One of the most important apps, responsible for processing and evaluating submissions made by participants. It contains code for creating a submission, changing the visibility of the submission and populating the leaderboard for any challenge.
 
+
```
Comment (deshraj) [inline docs/source/directory_structure.md]: No need for a newline here.

File: docs/source/evaluation_scripts.md [hunk 7]
```diff
@@ -6,6 +6,7 @@ The logic for evaluating and judging a submission is customizable and varies fro
 
 Evaluation scripts are required to have an `evaluate` function. This is the main function, which is used by workers to evaluate the submission messages.
 
+
```
Comment (deshraj) [inline docs/source/evaluation_scripts.md]: Please remove this change.

File: docs/source/setup.md [hunk 8]
```diff
@@ -45,7 +45,7 @@ instructions](https://help.github.com/articles/connecting-to-github-with-ssh/).
 
 * In your browser, visit [https://github.com/Cloud-CV/EvalAI](https://github.com/Cloud-CV/EvalAI) and click the `fork` button. You will need to be logged in to GitHub to do this.
 
-* Open Terminal and clone your fork by
+* Open Terminal and clone your fork by typing
```
Comment (deshraj) [inline docs/source/setup.md]: Please revert this change.

File: docs/source/setup.md [hunk 9]
```diff
@@ -157,7 +157,7 @@ instructions](https://help.github.com/articles/connecting-to-github-with-ssh/).
...
```
Comment (deshraj) [inline docs/source/setup.md]: Please revert this change.

File: docs/source/setup.md [hunk 10]
```diff
@@ -167,7 +167,7 @@ Don't forget to replace `YOUR_GITHUB_USER_NAME` with your git username.
 
 ### Step 3: Setup code base
 
-* Create a python virtual environment and install python dependencies.
+* Create a Python virtual environment and install Python dependencies.
```
Comment (deshraj) [inline docs/source/setup.md]: Please revert this change.

**Comment by deshraj** [comment]
Closing this due to inactivity.
